### PR TITLE
Updates main github

### PR DESCRIPTION
**Pull request by SlnPons**

### CODE REVIEW

**Comment by SlnPons** [comment]
Guys, if a builder from the community wants to participate to the project - how can they do? 
Did I remove too much?
(Lucas I didn't add the GCC installation for all OS to be consistent...feel free to correct this if it's wrong :/)

File: INSTALLATION.md [hunk 1]
```diff
@@ -33,18 +33,16 @@ Two steps are required to use our web on-chain product:
 * Thyra installation : obtain the Thyra binary corresponding to your operating system (OS), rename it and make it executable.
 * DNS configuration : install and configure your DNS to resolve the massa top level domain (*.massa) where Thyra runs.
 
-You can either perform these two steps automatically, using an installation script, or manually using the binaries and command lines.
+The following installation guide allows you to perform these steps automatically using a simply script to run on your terminal.
```
Comment (gregLibert) [inline INSTALLATION.md:36]:
```suggestion
The following installation guide allows you to perform these steps automatically using a simple script to run on your terminal.
```

**Review by gregLibert — CHANGES_REQUESTED** [review]
:mag:

**Review by gregLibert — APPROVED** [review]
:+1:

**Pull request merged**